diskpy/ICgen/_compatible_spline.py

```diff
@@ -33,9 +33,10 @@ def splfcn(xpts):
             
             xpts = np.asarray(xpts)
             yinterp = splobj(xpts)
-            mask = (xpts < bbox[0]) | (xpts > bbox[1])
-            yinterp[mask] = ext
+# JW hack
```

Comment:
@wadsley can you say why this needed to be commented?

Anyway, with newer versions of scipy it shouldn't matter.